pages/groups/group.js

```diff
@@ -561,6 +570,12 @@ class GroupProfile extends Component {
             </TabPanel>
           </div>
         </Main>
+
+        {Boolean(showRegistrationModal) && (
```

Comment:
The `Boolean` cast here is superfluous here since we only ever expect `showRegistrationModal` to be `true` or `false`

Reply:
Legit. I'll update.